packages/office-ui-fabric-react/src/components/Shimmer/examples/Shimmer.Application.Example.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
 import { BaseComponent } from 'office-ui-fabric-react/lib/Utilities';
 import { createListItems } from '../../../utilities/exampleData';
 import { IColumn, buildColumns, SelectionMode, Toggle } from 'office-ui-fabric-react/lib/index';
-import { ShimmeredDetailsList } from '../../DetailsList';
+import { ShimmeredDetailsList } from '../../DetailsList/ShimmeredDetailsList';
```

Comment:
@ThomasMichon This is in code examples and any user reading the documentation will see it... I don't believe this shows them how to appropriately import this component. Shouldn't it be `import { ShimmeredDetailsList } from 'office-ui-fabric-react/lib/ShimmeredDetailsList';`? Read my previous comment.

Reply:
I don't even remember why I used `../../DetailsList`